Assets/VRTK/Scripts/Locomotion/VRTK_TouchpadMovement.cs

```diff
@@ -0,0 +1,480 @@
+﻿// Touchpad Movement|Locomotion|20062
+namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Adds the ability to move and rotate play area by using the touchpad. 
+    /// </summary>
+    /// <remarks>
+    /// The Touchpad Movement script requires VRTK_BodyPhysics script to be present in one of the scene GameObjects for collision detection. 
+    /// Different movement types can be split across the controllers by having one script per side and with the desired options. 
+    /// Warp and snap rotate options may provide more comfortable experience for some and blink can be used to soften the effects. 
+    /// Snap rotate and flip direction options can be useful with teleport scripts for seated experiences and for people using front facing camera setups (Oculus default, PSVR). 
+    /// </remarks>
+    public class VRTK_TouchpadMovement : MonoBehaviour
+    {
+        /// <summary>
+        /// Movement types that can be performed by the vertical axis.
+        /// </summary>
+        public enum VerticalAxisMovement
+        {
+            /// <summary>
```

Comment:
Whilst this is the "correct" way of doing the enum docs. The current doc generator doesn't work like this.

There is a github issue to update all enums this way -> https://github.com/thestonefox/VRTK/issues/804

But this means we can either wait to merge this pull request until the doc generator has been done, or you can do the docs in the old (wrong) way and then when the pull issue above is done it will also update this script to be correct.